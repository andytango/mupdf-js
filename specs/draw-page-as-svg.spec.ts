import { forExamplePages, loadExampleFile } from "./helpers/example-file";

describe("drawPageAsSVG", () => {
  forExamplePages("should return an svg string", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.drawPageAsSVG(doc, page)).toMatchSnapshot();
  });
});
