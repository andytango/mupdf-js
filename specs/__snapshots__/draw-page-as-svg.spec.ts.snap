// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drawPageAsSVG Page [1 / 4] - should return an svg string 1`] = `"cea115816191b0ac3840082bf3202a80"`;

exports[`drawPageAsSVG Page [2 / 4] - should return an svg string 1`] = `"ccad4c4e2ad07dce915f3f39b3d3c644"`;

exports[`drawPageAsSVG Page [3 / 4] - should return an svg string 1`] = `"404694dd209f7c8702bfc018fc3497ac"`;

exports[`drawPageAsSVG Page [4 / 4] - should return an svg string 1`] = `"33119084554b40041b6fc768479217a3"`;
